### PR TITLE
Adding Microsoft Accessibility Insights

### PR DESCRIPTION
**Pull request by KevinLaMS**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/100854)

### CODE REVIEW

**Comment by ItzLevvie** [comment]
This pull request may require approval from the Accessibility Insights engineering team.

* Related to https://github.com/microsoft/winget-pkgs/issues/59885 by @DaveTryon

File: manifests/m/Microsoft/Accessibility/Insights/1.1.2213.01/Microsoft.Accessibility.Insights.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.Accessibility.Insights
+PackageVersion: 1.1.2213.01
+PackageLocale: en-US
+Publisher: Microsoft
+PackageName: Accessibility Insights For Windows v1.1
```
Comment (stephengillie) [inline manifests/m/Microsoft/Accessibility/Insights/1.1.2213.01/Microsoft.Accessibility.Insights.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://www.microsoft.com/
PublisherSupportUrl: https://accessibilityinsights.io/docs/general/support/
PrivacyUrl:   https://privacy.microsoft.com/en-us/privacystatement
PackageName: Accessibility Insights For Windows v1.1
PackageUrl: https://accessibilityinsights.io/downloads/
```

**Comment by stephengillie** [comment]
> This pull request may require approval from the Accessibility Insights engineering team.
> 
> * Related to [[Package Request]: Please stop packaging AccessibilityInsights releases #59885](https://github.com/microsoft/winget-pkgs/issues/59885) by @DaveTryon ([dtryon@microsoft.com](mailto:dtryon@microsoft.com))

I haven't approved this package because of this concern. @KevinLaMS, what are your thoughts?

**Comment by ItzLevvie** [comment]
cc @denelon

**Pull request closed without merging**